src/Common/Doctrine/Entity/Meta.php

```diff
@@ -124,6 +124,8 @@ public function __construct(
         bool $titleOverwrite,
         string $url,
         bool $urlOverwrite,
+        ?string $canonicalUrl,
```

Comment:
@jonasdekeukelaere could you maybe make a small PR for this here?
This is a BC break that I missed since it doesn't have default values

Reply:
A small PR for what?